ironfish-cli/src/commands/wallet/remove.ts

```diff
@@ -42,7 +48,11 @@ export class RemoveCommand extends IronfishCommand {
         this.exit(1)
       }
 
-      await client.removeAccount({ name, confirm: true })
+      if (wait) {
+        this.log(`Waiting for account deletion to finish . . .`)
+      }
```

Comment:
You should use oclif status spinner thing we use in other places to indicate a long action is being performed.  You'll need to just check what happens if it throws inbetween starting and ending the action though. It should be fine, but just throw a demo error and check the output is correct.

https://github.com/iron-fish/ironfish/blob/master/ironfish-cli/src/commands/chain/prune.ts#L80-L82